extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -214,7 +214,7 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, (env) => {
     });
 
     it('handles array with no vars', () => {
-      check('${array}', 'foo,bar', {
+      return check('${array}', 'foo,bar', {
```

Comment:
There are maybe other places where we fail to return the `check()` promise, could you double check?

Reply:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/test/test-variables.js#L121

Reply:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/test/test-variables.js#L298